packages/storybook8/stories/Components/SitePermissions/Docs.mdx

```diff
@@ -1,12 +1,15 @@
 import { Canvas, Description, Meta, Source } from '@storybook/blocks';
 import * as SitePermissionsStories from './index.stories';
+import { SingleLineBetaBanner } from '../../BetaBanners/SingleLineBetaBanner';
 import SitePermissionsCheckModalExample from '!!raw-loader!./snippets/SitePermissionsCheckModal.snippet.tsx';
 import SitePermissionsDeniedModalExample from '!!raw-loader!./snippets/SitePermissionsDeniedModal.snippet.tsx';
 import SitePermissionsExample from '!!raw-loader!./snippets/SitePermissionsExample.snippet.tsx';
 import SitePermissionsRequestModalExample from '!!raw-loader!./snippets/SitePermissionsRequestModal.snippet.tsx';
 
 <Meta of={SitePermissionsStories} />
 
+<SingleLineBetaBanner />
+
 # Site Permissions
```

Comment:
beta banner was missing for site permission, so added one line fix here @prabhjot-msft